Base/QTCore/qSlicerScriptedFileReader.cxx

```diff
@@ -239,6 +239,12 @@ QStringList qSlicerScriptedFileReader::extensions()const
   return extensionList;
 }
 
+//-----------------------------------------------------------------------------
+qSlicerIOOptions* qSlicerScriptedFileReader::options()const
+{
+  return nullptr;
```

Comment:
Describe here and/or in the header file that returning nullptr is not an error it simply means that there are no options for this reader.

Reply:
Additional updates will be pushed to effectively implements this. In the mean time, the PR is in draft.